homeassistant/components/ibeacon/config_flow.py

```diff
@@ -29,3 +38,58 @@ async def async_step_user(
             return self.async_create_entry(title="iBeacon Tracker", data={})
 
         return self.async_show_form(step_id="user")
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> OptionsFlow:
+        """Get the options flow for this handler."""
+        return OptionsFlow(config_entry)
+
+
+class OptionsFlow(config_entries.OptionsFlow):
+    """Handle options."""
+
+    def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input: dict | None = None) -> FlowResult:
+        """Manage the options."""
+        errors = {}
+
+        current_uuids = self.config_entry.options.get(CONF_ALLOW_NAMELESS_UUIDS, [])
+        new_uuid = None
+
+        if user_input is not None:
+            new_uuid = user_input.get("new_uuid", "").lower()
+            if new_uuid and not VALID_UUID.match(new_uuid):
+                errors["new_uuid"] = "invalid_uuid_format"
```

Comment:
It would be better to let stdlib do the validation by creating a UUID object 
https://docs.python.org/3/library/uuid.html#uuid.UUID

Reply:
Done.

Also added another test to reach the codecov target (we were below by some tiny number).